lib/internal/Magento/Framework/Encryption/Adapter/Mcrypt.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace Magento\Framework\Encryption\Adapter;
+
+class Mcrypt implements EncryptionAdapterInterface
+{
+    /**
+     * @var string
+     */
+    private $cipher;
+
+    /**
+     * @var string
+     */
+    private $mode;
+
+    /**
+     * @var string
+     */
+    private $initVector;
+
+    /**
+     * @var resource
+     */
+    private $handle;
+
+    /**
+     * Mcrypt constructor.
+     * @param string $key
+     * @param string $cipher
+     * @param string $mode
+     * @param bool $initVector
+     * @throws \Exception
+     */
+    public function __construct(
+        $key,
+        $cipher = MCRYPT_BLOWFISH,
+        $mode = MCRYPT_MODE_ECB,
+        $initVector = false
+    ) {
+        $this->cipher = $cipher;
+        $this->mode = $mode;
+        // @codingStandardsIgnoreStart
```

Comment:
With `@codingStandardIgnoreLine` the code will look cleaner, in case you need to ignore just one line.